tasks_management/validation.py

```diff
@@ -94,7 +95,7 @@ def validate_bf_unique_code(code, uuid=None):
 
 
 def validate_not_empty_field(string, field):
-    if not string:
+    if string.strip() == "":
         return [{"message": _("tasks_management.validation.field_empty") % {
             'field': field
         }}]
```

Comment:
I don't think this statement is correct. Strip without any input removes whitespace characters from left and right side, however in case of this case I don't think that's what we want. Name of the function suggest empty field and whitespace is character as any other. If you would like to ensure that no leading or trailing spaces are passed it should be done either on the fronted site or in some data cleaning step in the mutation, before CRUD service is used. Also this would raise AttributeError for None input, which in this context should be treated as empty. None == empty might vary depending on the use (in example if we want to check if list is empty before we put data in there None in place of list should actually fail process), however in this case no input should be treated same as "" input.

Reply:
I just check empty string now. Frontend makes sure it would not be something like: "      ".